modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -1259,7 +1259,7 @@ class AggregateExpr(BaseExpr):
     ----------
     agg : str
         Aggregate name.
-    op : BaseExpr
+    op : BaseExpr or list of BaseExpr
```

Comment:
as far I understand the `op` field logic it has to contain a columnar expression to perform an aggregation on it, not the aggregation's arguments. Maybe it would be better to separate these two?
```python
# expr = AggregationExpr("nlargest", [col_expr, n])
expr = AggregationExpr("nlargest", col_expr, args={"n": n})
```
Because in the current implementation passing several expressions would look ambiguous, as in this case, I would expect to perform summation over both of the columns, but an actual implementation would do something else:
```python
AggregationExpr("sum", [col1_expr, col2_expr])
```

Reply:
In case of TOP_K aggregation, this field should contain 2 columnar expressions:
```json
        {
          "agg": "TOP_K",
          "type": {
            "type": "DOUBLE",
            "nullable": true
          },
          "distinct": false,
          "operands": [
            1,
            2
          ]
        }
```
Before the aggregation, the argument is added as a new column in the table. The aggregate refers to both columns - to perform the aggregation on and the argument column.

Reply:
okay I see, looks like we're inheriting HDK's logic here